modules/tinymce/src/core/main/ts/api/Editor.ts

```diff
@@ -577,7 +578,7 @@ class Editor implements EditorObservable {
    *
    * @method queryCommandValue
    * @param {string} cmd Command to query value from.
-   * @return {Object} Command specific value, for example the current font size.
+   * @return {string} Command value, for example the current font size or an empty string (`""`) if the query command is not found.
```

Comment:
I wonder if we'd be better returning `null` or `undefined` here instead of an empty string, as a command might actually need to return an empty string here?

Reply:
It will only return the empty string on removed editors or for commands that doesn't exist and both of them could be detected by using the queryCommandSupported so checking if it returned null or not is probably not the right way to see if a command was supported or not. I thought it would be nice to remove the `string | undefined` and `boolean | undefined` types that is there now.
The browsers return an empty string if they are not supported but we are moving away from browser like functionality in this PR so we could switch them to null if we want that type I'm not sure what is the best thing for this edge case. I can switch it I don't mind either way as long as it's not `undefined` as it used to be for removed editors null seems a bit more explicit that we have picked a proper return not just what would be default on a function not returning anything.

Reply:
Right I forgot that the native implementation used to do that anyways. In that case I think it's fine, as it's what people would already expect. Anyways, happy to go with whatever you want to do here Spocke 👍